core/domain/email_manager_test.py

```diff
@@ -2696,18 +2696,25 @@ def test_assign_translation_reviewer_email_for_invalid_review_category(
             email_manager.send_email_to_new_community_reviewer(
                 self.translation_reviewer_id, 'invalid_category')
 
-    def test_all_category_keys_present_in_new_reviewer_email_data_constant(
-            self):
-        self.assertEqual(len(email_manager.NEW_REVIEWER_EMAIL_DATA), 3)
-        self.assertTrue(
-            constants.REVIEW_CATEGORY_TRANSLATION in (
-                email_manager.NEW_REVIEWER_EMAIL_DATA))
-        self.assertTrue(
-            constants.REVIEW_CATEGORY_VOICEOVER in (
-                email_manager.NEW_REVIEWER_EMAIL_DATA))
-        self.assertTrue(
-            constants.REVIEW_CATEGORY_QUESTION in (
-                email_manager.NEW_REVIEWER_EMAIL_DATA))
+    def test_schema_of_new_reviewer_email_data_constant(self):
+        self.assertEqual(sorted(email_manager.NEW_REVIEWER_EMAIL_DATA.keys()), [
+            constants.REVIEW_CATEGORY_QUESTION,
+            constants.REVIEW_CATEGORY_TRANSLATION,
+            constants.REVIEW_CATEGORY_VOICEOVER])
+
+        self.assertEqual(sorted(email_manager.NEW_REVIEWER_EMAIL_DATA[
```

Comment:
Don't do three separate checks. Do a loop through all keys to check the three usual things and to check that exactly one of rights_message and rights_message_template is contained.

That way, this validation will scale if more review categories are added, without you needing to modify the tests in the future.

Ditto below.

Reply:
Done!